### PR TITLE
Materialize error pages and announcement modal

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
**In this PR:**
- Replaced bootstrap modal with angular material dialog for announcements
- Replaced bootstrap classes with angular material components in error pages
- Renamed dialog template to reflect the service and directive that govern it
- Slightly adjusted some of the phrasing on error pages

**Notes:**
- Previously, the announcement `description` was limited to screen readers only and the image was hidden on small screens. It looks goofy to have a dialog with no content, so I set up the `description` to show when the image is hidden. Functionality on large screens is the same as it was before.

### Screenshots
screen shot 2016-09-26 at 11 01 25 am

screen shot 2016-09-26 at 11 01 52 am

screen shot 2016-09-23 at 1 20 23 pm

### CODE REVIEW

File: uw-frame-components/portal/main/partials/access-denied.html [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <frame-page app-title='{{NAMES.title}} - Access denied' white-background='true'>
-  <h2 class='center'>Sorry, you're not authorized to access this.</h2>
-  <p class='center'>
-    <i class='fa fa-exclamation-triangle fa-3x warning-color'></i>
+  <md-content layout-padding layout="column" layout-align="center center">
+    <h2 class="md-display-1">You are not allowed to accent this content</h2>
```
Comment (Doug-Reed) [inline uw-frame-components/portal/main/partials/access-denied.html]: You are not allowed to "accent"...?

Comment (thevoiceofzeke) [inline uw-frame-components/portal/main/partials/access-denied.html]: Oooops

**Comment by thevoiceofzeke** [comment]
@timlevett Hmm...maybe I'll mark this WIP and take that on next week. 

...unless of course someone else feels like taking care of it.

File: uw-frame-components/portal/features/partials/features-dialog-template.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<md-dialog aria-label="new features">
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <h1>{{ latestFeature.title }}</h1>
+      <span flex></span>
+      <md-button class="md-icon-button" ng-click="closeDialog('dismissed')" aria-label="close features dialog">
```
Comment (apetro) [inline uw-frame-components/portal/features/partials/features-dialog-template.html]: Good attention to `aria-label`.

File: uw-frame-components/portal/features/partials/features-dialog-template.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <md-icon>close</md-icon>
+      </md-button>
+    </div>
+  </md-toolbar>
+    <md-dialog-content>
+      <div class="md-dialog-content features-popup" layout="column" layout-align="center center">
+        <img ng-src="{{ latestFeature.img }}" aria-describedby="featureContent" hide-xs>
+        <p id="featureContent" class="sr-only">{{ latestFeature.description }}</p>
+        <p hide-gt-xs>{{ latestFeature.description }}</p>
```
Comment (apetro) [inline uw-frame-components/portal/features/partials/features-dialog-template.html]: At `xs` is a screen reader going to try to read `latestFeature.description` twice?

Comment (thevoiceofzeke) [inline uw-frame-components/portal/features/partials/features-dialog-template.html]: Hmm...yes. Brb making a change.

**Pull request merged**